community/contribution_guide/release.md

```diff
@@ -337,31 +351,35 @@ svn --username=${APACHE LDAP USERNAME} commit -m "release ${RELEASE.VERSION}"
 ### dev@seatunnel.apache.org Voting
 #### Voting initiate
 ```bash
-[VOTE] Release Apache SeaTunnel 2.3.3
+[VOTE] Release Apache SeaTunnel 2.3.8
```

Comment:
Lets start with release condidate 0
Change "[VOTE] Release Apache SeaTunnel 2.3.8" to "[VOTE] Release Apache SeaTunnel 2.3.8 condidate 0"

Reply:
+1

Reply:
how about start first vote with `[VOTE] Release Apache SeaTunnel 2.3.8 (RC1)`

Reply:
this is also ok. we can have RC1

Reply:
Done